src/lib/Makefile.linux

```diff
@@ -16,4 +16,8 @@ KBUILD_CFLAGS += -D__AARCH64__
 KBUILD_CFLAGS += -I$(src)/$(CASEMATELIB)
 KBUILD_CFLAGS += -I$(src)/$(CASEMATELIB)/include/
 
+ifneq ($(GHOST_USES_SERIAL),)
```

Comment:
Might want to eventually make this a bit more generic, and directly set some `-DCONFIG_LINUX -DCONFIG_HAS_pKVM_SERIAL_MODULE` or something, but later...